crates/libcontainer/src/rootless.rs

```diff
@@ -96,6 +97,26 @@ impl<'a> From<&'a Linux> for Rootless<'a> {
     }
 }
 
+#[cfg(not(test))]
```

Comment:
This is a super cool idea. Can you add some comments for future us on how this is used in the unit tests?

Reply:
I'm sorry but I couldn't imagine what kind of comments you want 🙇 Please let me more concreate.